### ISSUE TITLE
Create large dataset on random policy

### ISSUE THREAD

**Issue by aigagror** [aigagror/MuZeroGoJax#168]
Decreased logits for occupied spaces
